redis_setup.sh

```diff
@@ -1,5 +1,8 @@
 
 #It will remove all the previous docker engine
+docker -v
+if [ echo "$?" = "127" ]
```

Comment:
What is this?

Reply:
if there will be no docker in machine, the command `docker -v` will throw error `127`. But if docker is already present, the command will throw  `0`. 
So, it is just a check for docker. And same I have done in line 26, for check `redislabs/redisgraph` docker image.

Reply:
> #It will remove all the previous docker engine
Update this comment to explain this then we can merge this.